### PR TITLE
{Release} Upgrade to Azure CLI 2.8.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S171

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,163 @@
...
+* `az network vnet create`: support --nsg argument (#13842)
+* `az network lb address-pool`: support create lb backend pool with backend address. (#13700)
+* `az network application-gateway address-pool`: fix for --add argument (#14010)
+
+**RBAC**
+
+* `az ad sp create-for-rabc`: support name with space, slash and back slash (#13875)
+* `az ad sp create-for-rbac`: refine error message when user specify an invalid scope (#13117)
+
+**security**
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:142]:
```suggestion
**Security**
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,163 @@
...
+* `az appservice ase list-addresses`: Fix change of SDK #13140. (#13744)
+* Fix webapp/slot creation for Windows Containers (#13813)
+* `az webapp auth update`: Add optional parameter to update runtime-version (#13366)
+* Support list, delete, approve and reject private endpoint connection for webapp in CLI (#13710)
+* Fix #13888 : Add support for Static WebApps: get, list, create commands (#13639)
+* Improved error messages for SSH Tunnel Connection (#13997)
+
+**ARM**
+
+* Add examples for az tag -h (#10880)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:51]:
```suggestion
* `az tag`: Add examples for -h (#10880)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,163 @@
...
+* Support string/bytes encryption and decryption with stored keys (#13916)
+
+**Monitor**
+
+* Support no wait for cluster creation (#13787)
+* `az monitor log-analytics workspace saved-search`: Support new commands for saved search (#13816)
+
+**Network**
+
+* : Refine help message and add examples for az network application-gateway address-pool update (#13780)
```
Comment (MyronFanQiu) [inline src/azure-cli/HISTORY.rst:132]:
```suggestion
* `az network application-gateway address-pool update`: Refine help message and add examples. (#13780)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,163 @@
...
 ===============
 
+2.8.0
+++++++
+
+**ACR**
+
+* Add support for region endpoint disable / routing disable (#13617)
+* [BREAKING CHANGE] `az acr login --expose-token` does not accept username and password (#13874)
+* `az acr agentpool`: New CLI testing (#12914)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:13]:
```suggestion
* [BREAKING CHANGE] `az acr login --expose-token` does not accept username and password (#13874)
```
testing is not customer facing.

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,163 @@
...
+* CHANGE help text for "--expiry" parameter. (#13940)
+
+**App Service**
+
+* `az webapp log deployment show`: Show the latest deployment log, or the deployment logs of a specific deployment if deployment-id is specified (#13889)
+* `az webapp log deployment list`: List of deployment logs available (#13889)
+* Fix: Surface error when invalid webapp name provided (#13939)
+
+**AppService**
+
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:39]:
```suggestion
**AppService**

* `az webapp log deployment show`: Show the latest deployment log, or the deployment logs of a specific deployment if deployment-id is specified (#13889)
* `az webapp log deployment list`: List of deployment logs available (#13889)
* Fix: Surface error when invalid webapp name provided (#13939)
```

File: src/azure-cli/HISTORY.rst (hunk 4 above)
Comment (haroldrandom) [reply src/azure-cli/HISTORY.rst:13]: forget to add changes?

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,163 @@
...
+**Iot Hub**
+
+* [BREAKING CHANGE] az iot hub job: Remove deprecated job commands (#13955)
+
+**Keyvault**
+
+* 'az keyvault key import' now supports importing from strings via two new parameters. (#13771)
+
+**KeyVault**
+
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:119]:
```suggestion
**KeyVault**

* 'az keyvault key import' now supports importing from strings via two new parameters. (#13771)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,160 @@
...
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:119]:
```suggestion
**KeyVault**

* 'az keyvault key import' now supports importing from strings via two new parameters. (#13771)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,157 @@
...
+* `az cosmosdb`: add exists command to database and container groups (#12774)
+
+**CosmosDB**
+
+* Allow creating fixed collections (#13950)
+
+**eventhubs**
+
+* [][]: az eventhubs namespace create : add managed identity parameters (#13992)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:100]:
```suggestion
**EventHub**

* `az eventhubs namespace create` : add managed identity parameters (#13992)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,157 @@
...
+
+* `az eventhubs namespace create` : add managed identity parameters (#13992)
+
+**Extension**
+
+* Add --version to support to install from a specific version (#13789)
+
+**Extensions**
+
+* Enable CLI extensions to include packages in the 'azure' namespace (#13163)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:108]:
```suggestion
**Extension**

* Add --version to support to install from a specific version (#13789)
* Enable CLI extensions to include packages in the 'azure' namespace (#13163)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,153 @@
...
+* Add --version to support to install from a specific version (#13789)
+* Enable CLI extensions to include packages in the 'azure' namespace (#13163)
+
+**Iot Hub**
+
+* [BREAKING CHANGE] az iot hub job: Remove deprecated job commands (#13955)
+
+**KeyVault**
+
+* 'az keyvault key import' now supports importing from strings via two new parameters. (#13771)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:112]:
```suggestion
* `az keyvault key import`: Supports importing from strings via two new parameters. (#13771)
```

Comment (fengzhou-msft) [reply src/azure-cli/HISTORY.rst:112]: Use `Support`

File: src/azure-cli/HISTORY.rst [hunk 11]
```diff
@@ -3,6 +3,153 @@
...
+* Support no wait for cluster creation (#13787)
+* `az monitor log-analytics workspace saved-search`: Support new commands for saved search (#13816)
+
+**Network**
+
+* `az network application-gateway address-pool update`: Refine help message and add examples. (#13780)
+* `az network vnet create`: support --nsg argument (#13842)
+* `az network lb address-pool`: support create lb backend pool with backend address. (#13700)
+* `az network application-gateway address-pool`: fix for --add argument (#14010)
+
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:126]:
```suggestion
* `az network private-link-resource`: Add support for the Microsoft.Batch batchAccount resource (#13796)
* `az network private-endpoint-connection`: Add support for the Microsoft.Batch batchAccount resource (#13796)

```

File: src/azure-cli/HISTORY.rst [hunk 12]
```diff
@@ -3,6 +3,153 @@
...
+**Batch**
+
+* `az batch account create`: Added new parameter `--public-network-access` (#13796)
+* `az batch account create`: Added new parameter `--identity-type` (#13796)
+* `az batch account set`: Added new parameter `--identity-type` (#13796)
+* [BREAKING CHANGE] az batch pool create: When creating a pool using a custom image, the --image property of can now only refer to a Shared Image Gallery image. (#13796)
+* [BREAKING CHANGE] az batch pool create: When creating a pool with --json-file option and specifying a networkConfiguration, the publicIPs property has moved in to a new property publicIPAddressConfiguration. This new property also supports a new ipAddressProvisioningType property which specifies how the pool should allocate IP's and a publicIPs property which allows for configuration of a list of PublicIP resources to use in the case ipAddressProvisioningType is set to UserManaged (#13796)
+
+**Batch/Network**
+
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:67]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 13]
```diff
@@ -3,6 +3,150 @@
...
+* `az image builder customizer add`: Support Windows update customizer (#13130)
+* New command `az image builder cancel` (#13130)
+* Show a warning when a user deploys a VMSS pinned to a specific image version rather than latest (#14006)
+
+**Cosmos DB**
+
+* `az cosmosdb`: add exists command to database and container groups (#12774)
+
+**CosmosDB**
+
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:91]:
```suggestion

**CosmosDB**

```

File: src/azure-cli-core/HISTORY.rst [hunk 14]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.8.0
+++++++
+* Add new get command loader mechanism when loading CommandLoader (#13763)
```
Comment (jiasli) [inline src/azure-cli-core/HISTORY.rst:8]: Please state the detail of "new get command loader mechanism"

File: src/azure-cli/HISTORY.rst [hunk 15]
```diff
@@ -3,6 +3,150 @@
...
+* `az sql db ltr-policy/ltr-backup`: update/show long term retention policy, show/delete long term retention backups, restore long term retention backup (#12897)
+
+**Storage**
+
+* Fix authentication issue to support get token for --subscription (#13845)
+* `az storage remove`: Fix issue #13459 to raise exception for operation failure (#13961)
+* Fix issues #13012, #13632 and #13657 to remove unused arguments for generate-sas related commands (#13936)
+* `az storage logging update`: Add check for logging version (#13962)
+* `az storage blob show`: Add more properties for blob with track 2 SDK (#13920)
+* Fix issue 13708 to refine warning message for credential (#13963)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:144]:
```suggestion
* Fix #13708: Refine warning message for credential (#13963)
```

File: src/azure-cli-core/HISTORY.rst (hunk 14 above)
Comment (Juliehzl) [inline src/azure-cli-core/HISTORY.rst:8]:
```suggestion
* Add new get_command_loader() entry when loading CommandLoader (#13763)
```

Comment (haroldrandom) [reply src/azure-cli-core/HISTORY.rst:8]: * Add get_command_loader() entry to support to load customized CommandLoader

File: src/azure-cli/HISTORY.rst [hunk 16]
```diff
@@ -3,6 +3,147 @@
...
+
+**ARO**
+
+* Add Public, Private to params for help with ingress/apiserver visibility (#13550)
+
+**Batch**
+
+* `az batch account create`: Added new parameter `--public-network-access` (#13796)
+* `az batch account create`: Added new parameter `--identity-type` (#13796)
+* `az batch account set`: Added new parameter `--identity-type` (#13796)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:62]:
```suggestion
* `az batch account create`: Add new parameter `--public-network-access` (#13796)
* `az batch account create`: Add new parameter `--identity-type` (#13796)
* `az batch account set`: Add new parameter `--identity-type` (#13796)
```

File: src/azure-cli/HISTORY.rst [hunk 17]
```diff
@@ -3,6 +3,147 @@
...
+
+* `az image builder`: Upgrade API version to 2020-02-14 (#13130)
+* `az image builder create`: Add `--identity` to support identity configuration (#13130)
+* `az image builder customizer add`: Support Windows update customizer (#13130)
+* New command `az image builder cancel` (#13130)
+* Show a warning when a user deploys a VMSS pinned to a specific image version rather than latest (#14006)
+
+**Cosmos DB**
+
+* `az cosmosdb`: add exists command to database and container groups (#12774)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:88]:
```suggestion
* `az cosmosdb`: Add exists command to database and container groups (#12774)
```

File: src/azure-cli/HISTORY.rst [hunk 18]
```diff
@@ -3,6 +3,147 @@
...
+* Allow creating fixed collections (#13950)
+
+**EventHub**
+
+* `az eventhubs namespace create` : add managed identity parameters (#13992)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:93]:
```suggestion
* `az eventhubs namespace create` : Add managed identity parameters (#13992)
```

File: src/azure-cli/HISTORY.rst [hunk 19]
```diff
@@ -3,6 +3,147 @@
...
+
+**Monitor**
+
+* Support no wait for cluster creation (#13787)
+* `az monitor log-analytics workspace saved-search`: Support new commands for saved search (#13816)
+
+**Network**
+
+* `az network application-gateway address-pool update`: Refine help message and add examples. (#13780)
+* `az network vnet create`: support --nsg argument (#13842)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:117]:
```suggestion
* `az network vnet create`: Support --nsg argument (#13842)
```

File: src/azure-cli/HISTORY.rst [hunk 20]
```diff
@@ -3,6 +3,147 @@
...
+* `az network lb address-pool`: support create lb backend pool with backend address. (#13700)
+* `az network application-gateway address-pool`: fix for --add argument (#14010)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:119]:
```suggestion
* `az network vnet create`: Support --nsg argument (#13842)
* `az network lb address-pool`: Support create lb backend pool with backend address. (#13700)
* `az network application-gateway address-pool`: Fix for --add argument (#14010)
```

File: src/azure-cli/HISTORY.rst [hunk 21]
```diff
@@ -3,6 +3,147 @@
...
+
+**RBAC**
+
+* `az ad sp create-for-rabc`: support name with space, slash and back slash (#13875)
+* `az ad sp create-for-rbac`: refine error message when user specify an invalid scope (#13117)
```
Comment (haroldrandom) [inline src/azure-cli/HISTORY.rst:124]:
```suggestion
* `az ad sp create-for-rabc`: Support name with space, slash and back slash (#13875)
* `az ad sp create-for-rbac`: Refine error message when user specify an invalid scope (#13117)
```

**Pull request merged**